src/platform/platform_posix.c

```diff
@@ -270,22 +270,23 @@ CxPlatRefIncrement(
 
 BOOLEAN
 CxPlatRefIncrementNonZero(
-    _Inout_ volatile CXPLAT_REF_COUNT* RefCount
+    _Inout_ volatile CXPLAT_REF_COUNT* RefCount,
+    _In_ uint32_t Bias
```

Comment:
For adding more than 1 ref at a time?

Reply:
This particular change was just to make all platforms have the same signature.